src/algebra/ordered_monoid.lean

```diff
@@ -208,14 +208,19 @@ namespace units
 instance [monoid α] [preorder α] : preorder (units α) :=
 preorder.lift (coe : units α → α)
 
-@[simp, to_additive, norm_cast]
+@[simp, norm_cast, to_additive]
 theorem coe_le_coe [monoid α] [preorder α] {a b : units α} :
   (a : α) ≤ b ↔ a ≤ b := iff.rfl
 
-@[simp, to_additive, norm_cast]
+-- should `to_additive` do this?
```

Comment:
@alexjbest Is this addressed by #5487 ? (I don't think this PR should have to wait for that one.)